instagram/src/components/LoginPage/LoginPage.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from "react";
+
+class LoginPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      login: "",
+      password: "",
+      loggedIn: false,
+      correctLogin: "anonuser",
+      correctPassword: "abc123"
+    };
+  }
+
+  loginInputHandler = event => {
+    this.setState({ login: event.target.value });
+  };
+
+  passwordInputHandler = event => {
+    this.setState({ password: event.target.value });

```

Comment:
2 approaches to make your inputhandlers DRYer and more reusable

One liner
```suggestion
    inputChangeHandler = event => this.setState({[event.target.name]: event.target.value})
```

Using destructuring
```suggestion
    inputChangeHandler = event => {
    const {name, value} = event.target
    this.setState({[name]: value})
  }
```